src/lib/utils.h

```diff
@@ -84,6 +84,36 @@ CK_RV utils_ctx_wrap_objauth(twist wrappingkey, twist objauth, twist *wrapped_au
  */
 CK_RV ec_params_to_nid(CK_ATTRIBUTE_PTR ecparams, int *nid);
 
+/**
+ * Applies a PKCS5 padding to a blocksize of 16.
```

Comment:
It is strange to document "PKCS5" while the function is named "PKCS7". Maybe you could make this more uniform?